crates/oxc_linter/src/rules/no_dupe_keys.rs

```diff
@@ -83,6 +77,7 @@ fn test() {
         ("var x = { a: 1, b: { a: 2 } };", None),
         ("var x = ({ null: 1, [/(?<zero>0)/]: 2 })", None),
         ("var {a, a} = obj", None),
+        // Our parser doesn't allow the '0' prefixed octal literals.
```

Comment:
It's not our parser, it's SyntaxError ;-)

Reply:
fixed in [a258e48](https://github.com/Boshen/oxc/pull/188/commits/a258e48d581bbc2da63743fd88dbb9095690e1ac)